src/server/components/feedback/controllers.ts

```diff
@@ -1,50 +1,59 @@
-import { NextFunction, Request, Response } from "express";
-import { createFeedback } from "server/models/feedback";
-import { FeedbackJsonData } from "server/models/types";
-import {
-  WebhookData,
-  formRunnerPostRequestSchema,
-} from "server/components/formRunner";
-
-export async function feedbackIngest(
-  req: Request,
-  res: Response,
-  next: NextFunction
-): Promise<void> {
+import type { Request, Response } from "express";
+import { formRunnerPostRequestSchema } from "server/components/formRunner";
+import type { WebhookData } from "server/components/formRunner";
+import { sendContactUsEmail } from "server/services/govuk-notify";
+import { logger } from "server/services/logger";
+
+export async function feedbackIngest(req: Request, res: Response) {
   const { value, error } = formRunnerPostRequestSchema.validate(req.body);
 
-  if (error !== undefined) {
-    res.status(400).json({ error: error.message });
+  if (error) {
+    logger.error(`${req.originalUrl} validation failed ${error.message}`);
+    res.status(400).json({ error: error.message }).send();
     return;
   }
 
-  const questionsAndAnswers: FeedbackJsonData["questionsAndAnswers"] =
-    value.questions.reduce(
-      (
-        acc: FeedbackJsonData["questionsAndAnswers"],
-        question: WebhookData["questions"][0]
-      ) => {
-        const { fields } = question;
-
-        fields.forEach((field) => {
-          const { title = field.key, answer } = field;
-          acc.push({ question: title, answer });
-        });
-
-        return acc;
-      },
-      []
-    );
+  const personalisation = formatMessage(value);
 
   try {
-    await createFeedback({
-      type: "serviceFeedback",
-      jsonData: {
-        questionsAndAnswers,
-      },
-    });
-    res.json({ success: true });
+    await sendContactUsEmail(personalisation);
+    return res.status(200).json({ success: true }).send();
   } catch (error) {
-    next(error);
+    logger.error(`feedbackIngest Error: ${error.errors ?? error.message}`);
+    res.status(400).json({ error: error.message }).send();
   }
 }
+
+function formatMessage(webhookData: WebhookData) {
+  const data: string[] = [];
+
+  let serviceType = "";
+  let country = "";
+
+  const fields = webhookData.questions.flatMap((question) => question.fields);
+
+  fields.forEach((field) => {
+    let answer = field.answer;
+
+    if (field.key === "serviceType") {
+      serviceType = field.answer;
+    }
+    if (field.key === "country") {
+      country = field.answer;
+    }
+    if (field.key === "reason") {
+      answer = `To ${field.answer ?? "Not supplied"}`;
+    }
+
+    data.push(`##${field.title}
+       ${answer}
```

Comment:
For this code is it possible that both `field.answer` and `field.key: "reason"` are both null or undefined?

Reply:
it's possible since field.answer can be null, but it's a required field so it won't be in this case. thanks for catching!